src/components/Person.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">
+        {age && `I am ${age}`}
+      </p>
+      <p className="Person__partner">
+        {(sex === 'm' && isMarried && `My wife's name is ${partnerName}`)
```

Comment:
You can move up logic, where you checking person sex, and simplify code in this tag.